src/components/WwwFrame/TheHeader.vue

```diff
@@ -600,13 +599,6 @@ export default {
 			this.profilePic = data?.my?.lender?.image?.url ?? '';
 			this.profilePicId = data?.my?.lender?.image?.id ?? null;
 			this.basketState = data || {};
-
-			// GROW-280 redirect to login instead of popup login experiment
-			const redirectToLoginExperiment = this.apollo.readFragment({
-				id: 'Experiment:redirect_to_login',
-				fragment: experimentVersionFragment,
-			}) || {};
-			this.redirectToLoginExperimentVersion = redirectToLoginExperiment.version;
```

Comment:
Could you please also remove the other references to `redirectToLoginExperimentVersion` in this file?

Reply:
Sure!